instrumentation/jedis/jedis-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v1_4/JedisConnectionInstrumentation.java

```diff
@@ -46,6 +46,20 @@ public void transform(TypeTransformer transformer) {
             .and(takesArgument(0, named("redis.clients.jedis.Protocol$Command")))
             .and(takesArgument(1, is(byte[][].class))),
         this.getClass().getName() + "$SendCommandWithArgsAdvice");
+    // For version 2.7.2
+    transformer.applyAdviceToMethod(
+        isMethod()
+            .and(named("sendCommand"))
+            .and(takesArguments(1))
+            .and(takesArgument(0, named("redis.clients.jedis.ProtocolCommand"))),
```

Comment:
you could use `.and(takesArgument(0, namedOneOf("redis.clients.jedis.Protocol$Command", "redis.clients.jedis.ProtocolCommand")))` in the original advice so you wouldn't need to duplicate this for 2.7.2